ArduPlane/quadplane.cpp

```diff
@@ -443,6 +443,15 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("ACRO_YAW_RATE", 13, QuadPlane, acro_yaw_rate, 90),
+
+    // @Param: LOIT_REV_SPD
+    // @DisplayName: QLOITER mode max reverse speed
+    // @Description: maximum backward airspeed in m/sec
```

Comment:
There's a discrepancy between the description and what it does. The description implies all ground speed (particularly since thats what the other speed limits are), but it's actually a function of airspeed which this isn't informing the user of.